src/vaults/factories/SyncDepositVaultFactory.sol

```diff
@@ -39,15 +41,15 @@ contract SyncDepositVaultFactory is Auth, IVaultFactory {
         address asset,
         uint256 tokenId,
         IShareToken token,
-        address, /* escrow */
         address[] calldata wards_
     ) public auth returns (address) {
         SyncDepositVault vault = new SyncDepositVault(
-            poolId, scId, asset, tokenId, token, root, syncRequests, asyncRequests, poolEscrowProvider
+            poolId, scId, asset, tokenId, token, root, syncDepositManager, asyncRedeemManager, poolEscrowProvider
         );
 
         vault.rely(root);
-        vault.rely(syncRequests);
+        vault.rely(address(syncDepositManager));
+        vault.rely(address(asyncRedeemManager));
```

Comment:
I'm pretty sure we need to add here `asyncRedeemManager` due asyncRedeemManager ends up using auth methods from the vault

Reply:
@lemunozm those reky should be Done in addVault. Can you Check? Or in approve Manager?

Reply:
@mustermeiszer We are doing the other direction in `addVault`: https://github.com/centrifuge/protocol-v3/blob/818362323b53de92b45aebd50023580bcc31c483/src/vaults/SyncRequests.sol#L116

AFAICT, both are necessary. So good catch @lemunozm on the missing rely of the async redeem manager.